### PR TITLE
MySQL Performance when using GF_DATABASE_URL

### LINKED ISSUE #9784: MySQL Performance when using GF_DATABASE_URL
Please include this information:
- What Grafana version are you using?
grafana:4.6.0-beta3 I can't use anything newer because I need alerts fixed in #9777
- What datasource are you using?
prometheus 2.0.0-rc.2
- What OS are you running grafana on?
docker
- What did you do?
loading the Home page
- What was the expected result?
take under 5 seconds
- What happened instead?
took 20 seconds
- If related to metric query / data viz:
  - Include raw network request & response: get by opening Chrome Dev Tools (F12, Ctrl+Shift+I on windows, Cmd+Opt+I on Mac), go the network tab. 
image
- config
Using env variable to point to a remote mysql db via GF_DATABASE_URL

I tried restarting the mysql and the grafana, with no change. I ran grafana from my development box with GF_DATABASE_URL and the home page loaded fast which made me think it was an issue with the docker. 

Then I enabled SQL debugging in Grafana for the database and found queries against the user table were taking 5 secs. I only have 2 users in grafana.
>"[SQL] SELECT
		u.id             as user_id,
		u.is_admin       as is_grafana_admin,
		u.email          as email,
		u.login          as login,
		u.name           as name,
		u.help_flags1    as help_flags1,
		u.last_seen_at   as last_seen_at,
		org.name         as org_name,
		org_user.role    as org_role,
		org.id           as org_id
		FROM `user` as u
		LEFT OUTER JOIN org_user on org_user.org_id = 1 and org_user.user_id = u.id
		LEFT OUTER JOIN org on org.id = org_user.org_id WHERE u.id=? [1] - took: 5.020826667s" logger=sqlstore.xorm

Then I tried using GF_DATABASE_TYPE, GF_DATABASE_HOST, GF_DATABASE_NAME, GF_DATABASE_USER, GF_DATABASE_PASSWORD variables instead of GF_DATABASE_URL and the docker was blazing fast. So I kept digging and found that when using the GF_DATABASE_URL config var grafana doesn't set the MaxOpenConn and MaxIdleConn like it does when the DATABASE configs are separate.
https://github.com/grafana/grafana/blob/c5400ffe76da20747252420df0d96a3738265610/pkg/services/sqlstore/sqlstore.go#L193
So I tried adding the MaxOpenConn and MaxIdleConn options to the GF_DATABASE_URL section and sure enough the home page loads fast again. 

Now it is taking under a second to load in the docker, when running with GF_DATABASE_URL and my change to add MaxOpenConn and MaxIdleConn 
image

### PR DESCRIPTION
**Pull request by dfredell**
Fix #9784 
Set MaxOpenConn and MaxIdleConn when using GF_DATABASE_URL

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -82,6 +82,9 @@ max_idle_conn = 2
 # Max conn setting default is 0 (mean not set)
 max_open_conn =
 
+# Set to true to log the sql calls and execution times.
+debug =
```
Comment (bergquist) [inline conf/defaults.ini]: I think we can give this setting a better name. How about `log_sql_queries` or `log_queries`?

**Review by bergquist — CHANGES_REQUESTED** [review]
I would the debug setting to be more declarative. Looks good otherwise!

File: conf/defaults.ini (hunk 1 above)
Comment (dfredell) [reply conf/defaults.ini]: Great plan, I like the more descriptive name. It is already in the database section so I think we can leave off the 'sql', as there wouldn't be any other 'queries' we are talking about.

**Comment by dfredell** [comment]
Great plan, I like the more descriptive name. It is already in the database section so I think we can leave off the 'sql', as there wouldn't be any other 'queries' we are talking about. I updated it to  GF_DATABASE_LOG_QUERIES.

**Review by bergquist — APPROVED** [review]

**Comment by bergquist** [comment]
`log_queries` makes sense :)

Thank you for contributing!

**Pull request merged**